.github/workflows/generate_pot.yml

```diff
@@ -0,0 +1,42 @@
+name: Generate translation template
+
+on:
+  pull_request:
+    paths-ignore:
+      - "*.md"
+    types:
+      - opened
+      - synchronize
+      - reopened
```

Comment:
Any reason the workflow triggers for pull requests? I don't think we want to generate translation templates for unmerged PRs and the `push` trigger already covers any merged PRs.

Reply:
I think there is value in running the workflow for unmerged pull requests, since I've added the `--check=space-ellipsis` xgettext option. That option makes the command fail if there are any translatable strings that contain spaces before ellipsis (which is considered a punctuation mistake). This way we can see and fix those errors before merging.
Obviously we can also avoid adding this kind of "lint" checks (or add other checks that are listed in the xgettext man page), the decision is up to you :)

Reply:
Alright, we can give this a try and see if it catches any such mistakes. Everything else looks good so I'll merge this now.